packages/frontend-app-api/src/wiring/createApp.tsx

```diff
@@ -415,6 +416,16 @@ function createApiHolder(
       ?.map(e => e.instance?.getData(coreExtensionData.theme))
       .filter((x): x is AppTheme => !!x) ?? [];
 
+  const translationResources =
+    tree.root.edges.attachments
```

Comment:
Could `attachments` now be renamed to `resolvedInputs`? Just thinking that maybe this is the only place that the word "attachment" basically is used? 🤔 Hm not sure. Maybe attachments is better still.

Reply:
Yep I think that's likely a good call 👍 , adding it to the meta issue

Reply:
Thought about this some more and it's probably still good to have entirely dedicated wording for the slot and the thing that goes into the slot. So input and attachment being different might be a good thing ... the things being put into the input aren't really "resolved inputs", per se - maybe it's that type that instead could be renamed to like `AttachmentSlot` or whatever just to distinguish it as "the type whose shape is purpose built to represent this attachment/attachments depending on what the input required". Well, some other day, when we get to that point.